webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -56,32 +57,33 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
             experimentId: parseInt(task.id),
             state: RunState.Active,
           });
-          await updateExperimentLocally(exp => ({ ...exp, state: RunState.Active }));
+          updateExperimentLocally(exp => ({ ...exp, state: RunState.Active }));
```

Comment:
question: should `updateExperimentLocally` be defined as an `async` function and have `await` applied to `setExperiments`?